packages/kbn-server-route-repository-utils/src/typings.ts

```diff
@@ -133,9 +101,9 @@ type ClientRequestParamsOfType<TRouteParamsRT extends RouteParamsRT> =
     ? MaybeOptional<{
         params: t.OutputOf<TRouteParamsRT>;
       }>
-    : TRouteParamsRT extends z.Schema
+    : TRouteParamsRT extends z.ZodSchema
```

Comment:
Can you briefly explain how `ZodSchema` and `ZodType` differ from `Schema`? 
I'm guessing something about them make the optional parts easier to manage since I don't see any other changes.

Reply:
AFAIK they're just aliases